apis/capabilities/v1alpha1/tenant_types.go

```diff
@@ -39,7 +41,8 @@ type TenantSpec struct {
 	SystemMasterUrl        string             `json:"systemMasterUrl"`
 	TenantSecretRef        v1.SecretReference `json:"tenantSecretRef"`
 	PasswordCredentialsRef v1.SecretReference `json:"passwordCredentialsRef"`
-	MasterCredentialsRef   v1.SecretReference `json:"masterCredentialsRef"`
+	// Namespace of the existing APIManager that you wish to used to create the Tenant
+	APIManagerNamespaceRef string `json:"apiManagerNamespaceRef"`
```

Comment:
This is what makes me feel uneasy. We are breaking backward compat. From the product's point of view it is ok, as this is still in tech preview. However, if there is a tenant CR created in a cluster and the operator is upgraded to a new version with a new Tenant CRD, what happens?

Reply:
We sort of knew there would be breaking changes. Should we add back in MasterCredentialsRef and make it optional and handle the logic with either scenario and update the docs to use APIManagerNamespaceRef only?